.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - 10
```

Comment:
While at that, let's also add `8`, `12`, and `node` so that we can safely know if there is a potential breakage with those different Node.js versions.

Reply:
Done.
Rebased with `upstream/master` too.
Travis result can be seen [here](https://travis-ci.com/gsukanto/universal-update-checker/builds/142094548).